### PR TITLE
Render the used config values as part of formatting examples

### PR DESCRIPTION
**Pull request by mads-hartmann**
This is a fix for #1130

### CODE REVIEW

**Comment by mads-hartmann** [comment]
It looks like this (where the config can be shown/hidden by clicking on the sentence)

screen shot 2018-05-02 at 22 13 21

File: readme/src/main/scala/org/scalafmt/readme/Readme.scala [hunk 1]
```diff
@@ -100,19 +101,50 @@ object Readme {
...
+    //    b = 1
+    //    c = 2
+    //   ]
+    // turns into
+    //   a.b = 1
+    //   a.c = 2
+    // which means a simple line-by-line string diffing is sufficient for now.
+    val defaultStrs = Config.toHocon(ScalafmtConfig.default).toSet
+    val configStrs = Config.toHocon(style).toSet
+    configStrs.diff(defaultStrs).mkString("\n")
```
Comment (olafurpg) [inline readme/src/main/scala/org/scalafmt/readme/Readme.scala]: That's pretty cool, amazed it works 👍

File: readme/src/main/scala/org/scalafmt/readme/Readme.scala [hunk 2]
```diff
@@ -100,19 +101,50 @@ object Readme {
...
+  }
+
+  def configurationBlock(style: ScalafmtConfig) = {
+    div(
+      span(
+        "Show/hide configuration used for this example",
+        `class` := "scalafmt-configuration-toggle"),
+      pre(changedConfig(style)),
+      `class` := "scalafmt-configuration collapsed"
```
Comment (olafurpg) [inline readme/src/main/scala/org/scalafmt/readme/Readme.scala]: Is it possible to have it expanded by default or does that result in too much bloat?

File: readme/src/main/scala/org/scalafmt/readme/Readme.scala (hunk 1 above)
Comment (mads-hartmann) [reply readme/src/main/scala/org/scalafmt/readme/Readme.scala]: Me too 😂

**Pull request merged**